### ISSUE TITLE
I think there is a small mistake in common.camera.py

### ISSUE THREAD

**Issue by ZaVang** [facebookresearch/VideoPose3D#202]
Thanks for your great work. But I read the code and find there might be a small mistake.
In the file `common.camera.py` there is a function `project_to_2d(X, camera_params)`:
    tan = torch.sum(p*XX, dim=len(XX.shape)-1, keepdim=True)
    XXX = XX*(radial + tan) + p*r2
I think the 'tan' variable should be '2*tan'. I don't know whether it effect the performance significantly, but it' really wrong. Would you like to check it again?
